benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -1201,6 +1201,7 @@
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "unprocessedPostponementRequest", "UserRole": "caseworker-sscs", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "actionPostponementRequestSelected", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "actionPostponementRequestSelected", "UserRole": "caseworker-sscs-clerk", "CRUD": "CRUD"},
+  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "actionPostponementRequestSelected", "UserRole": "caseworker-sscs-judge", "CRUD": "CRUD"},
```

Comment:
All these changes should be in the non prod sheets

Reply:
it is fine

Reply:
It does need to be in the non prod sheet. @filippo-possenti-version1

Reply:
Done.